src/physics/components.rs

```diff
@@ -107,10 +110,30 @@ pub struct PhysicsInterpolationComponent(pub Isometry<f32>);
 
 impl PhysicsInterpolationComponent {
     /// Create a new PhysicsInterpolationComponent from a translation and rotation
+    #[cfg(feature = "dim2")]
+    pub fn new(translation: Vec2, rotation: f32) -> Self {
```

Comment:
I'm not sure if this is going to be the right API for the 2D rotation, though it makes sense to me to just pass in an angle (rotation about z, right-handed, y-up, positive z out of the screen, rotation goes anti-clockwise as you look along negative z) and convert to a Complex using x (real) = 1 * cos(angle), y (imaginary) = 1 * sin(angle).

Reply:
Something like this could have worked but `RigidBody.joint_graph_index` and `JointSet.remove_rigid_body()` are crate-private:
```rust
    for entity in &joints_removed.clone() {
        if let Some((joint, entity1, entity2)) = entity_maps.joints.get(entity) {
            if let Some(rbh1) = entity_maps.bodies.get(entity1) {
                if let Some(rb1) = bodies.get(*rbh1) {
                    joints.remove_rigid_body(rb1.joint_graph_index, &mut bodies);
                }
            }
            if let Some(rbh2) = entity_maps.bodies.get(entity2) {
                if let Some(rb2) = bodies.get(*rbh2) {
                    joints.remove_rigid_body(rb2.joint_graph_index, &mut bodies);
                }
            }
            joints_removed.remove(entity);
            entity_maps.joints.remove(entity);
        }
    }
```

Reply:
This is a good idea. I suggest:
- Naming the parameter `rotation_angle`.
- Simply using `UnitComplex::new(rotation_angle)` instead of doing the sin/cos computation yourself.

Reply:
Done.